packages/sdk/src/actions/simulateUpdateDepositTx.ts

```diff
@@ -11,7 +11,7 @@ export type SimulateUpdateDepositTxParams = {
   originChainClient: ConfiguredPublicClient;
   destinationChainClient: ConfiguredPublicClient;
   deposit: {
-    depositId: number;
+    depositId: bigint;
```

Comment:
So this is a breaking change right? Maybe it makes sense to allow `number  | bigint` here and convert to a bigint in the function logic